sdk/storage/azure-storage-blob/pom.xml

```diff
@@ -151,6 +151,18 @@
       <version>1.7.3</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Noticed that JUnit tests are not run due to missing Jupiter engine

Reply:
I actually didn't even notice this until you pointed it out, but in Storage, we request that all tests be written in Groovy using Spock, not JUnit. If you would rather not convert them, then we can leave this open for now and I can do that when I get around to it.

Reply:
Oh I see, I saw a mix of JUnit / Spock across many modules. thought this is fine. I think it I could do it Spock style, sure

Reply:
Done, thanks @rickle-msft